### ISSUE TITLE
Check capture vs endgame graphs

### ISSUE THREAD

**Issue by sprunk** [ZeroK-RTS/Zero-K#4997]
IIRC there was some weirdness related to captured units. Death of a captured unit should probably count against the original owner, not the temporary.

**Comment by GoogleFrog** [comment]
Doesn't it all work out if capture counts as a death and creation?

**Comment by sprunk** [comment]
Not necessarily, what if a unit changes hands multiple times? "You lost a thousand metal" factoid actually just statistical error. Glaives georg, who lives under domi rule and changed hands 15 times, is an outlier adn should not have been counted.

But sure, looking at various alternatives and deciding that failing at edge cases X is still better than the alternative Y which fails at Z is a valid answer to the ticket.

**Comment by GoogleFrog** [comment]
Ah. I was thinking about the current value graph. You are thinking about cumulative losses. Capture would seem to cause problems for the cumulative graphs.
